python/src/FittingTest_doc.i.in

```diff
@@ -305,6 +311,10 @@ BIC : float
 Notes
 -----
 This is used for model selection.
+
+If a :class:`~openturns.Distribution` is used, the likelihood is evaluated on the sample. 
+If a :class:`~openturns.DistributionFactory` is used, the `build` method is used to create the distribution, and the likelihood is then evaluated.
+
```

Comment:
I think this can be moved below the "model" argument description

Reply:
Agree with @jschueller remark. IMO Notes are here to give further insight into the methods, algorithms or to expand the description already given in the Parameter section

Reply:
Ok. Fixed.